force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -34,19 +34,90 @@ public with sharing class DummyJSONCallout {
      * @param dummyUserId The ID of the user in the external system to fetch data for.
      */
 
-	public static void getDummyJSONUserFromId(String dummyUserId) {
-		// Create HTTP request to send.
-
-		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
-		// Set the HTTP method to GET.
-
-		// Send the HTTP request and get the response.
-
-		// If the HTTP response code is successful, parse the JSON response and update the contact.
-		Contact cont = null;
-		upsert cont DummyJSON_Id__c; //insert/update from the JSON response using the external id (dummyUserId)
-	}
+	 public static void getDummyJSONUserFromId(String dummyUserId) {
+		Http http = new Http();
+		HttpRequest request = new HttpRequest();
+		
+		// Set the endpoint URL
+		String endpoint = 'https://dummyjson.com/users/' + dummyUserId;
```

Comment:
Make you know how to use Named Credentials. That is considered best practice and you will need it for the capstone.